orchagent/orchdaemon.cpp

```diff
@@ -654,3 +663,50 @@ bool OrchDaemon::warmRestartCheck()
     gSwitchOrch->restartCheckReply(op,  data, values);
     return ret;
 }
+
+void
+OrchDaemon::check_and_exit()
+{
+    SWSS_LOG_ENTER();
+
+    swss::SubscriberStateTable tbl(m_configDb, "CONTAINER_FEATURE");
```

Comment:
make a constant fro that container feature?

Reply:
I don't get this comment. Can you please elaborate ?